default-react-modules/src/components/modules/BlogPosts-V0/index.jsx

```diff
@@ -674,15 +618,29 @@ const BlogPost = ({
       style={{ ...blogPostWrapperStyles, ...layoutStyle }}
     >
       {showPostImage && featuredImage && (
-        <BlogPostImage
-          blogPost={blogPost}
-          layout={layout}
-          imageStyle={imageStyle}
-          groupDefaultText={groupDefaultText}
-          loopIndex={loopIndex}
-          hasAlternatingImage={hasAlternatingImage}
-          firstRowImage={loopIndex < columns}
-        />
+        // There is currently no way to add classnames to
+        // Islands. Added this wrapper div so we can latch
+        // on to the media query styles in styles.module.css.
+        // We can move them onto the island when that is an option.
+        <div
+          style={imageWrapperStyles}
+          data-testid="blog-post-image-wrapper"
+          className={
+            layout === 'sideBySide'
+              ? `${styles.hsBlogPostImageWrapper} ${styles.hsBlogPostImageWrapperSideBySide}`
+              : `${styles.hsBlogPostImageWrapper}`
+          }
+        >
```

Comment:
FYI, you can use `wrapperClassName` on Islands now ([as of 0.14.6](https://github.com/HubSpot/cms-react/blob/e27b5a1298a77e174683e98189fe12589722c51b/docs/release-log.md?plain=1#L49-L54))

Reply:
Created an issue internally to make that update